docs/articles/documentation/guides/basic-guides/select-page-elements.md

```diff
@@ -0,0 +1,368 @@
+---
+layout: docs
+title: Select Page Elements
+permalink: /documentation/guides/basic-guides/select-page-elements.html
+---
+# Select Page Elements
+
+TestCafe needs to identify the target page element to perform an [action](interact-with-the-page.md) with it (click, drag, etc.) or obtain its state and check it in an [assertion](assert.md).
+
+You can pass a CSS selector to an action function to specify the target element:
+
+```js
+await t.click('#my-button');
+```
+
+For more complex queries, however, CSS selectors get longer and difficult to read, write, and maintain:
+
+```js
+await t.click('div > .my-class > div:nth-child(2) > span > a[href="https://my-site.com/page"]');
+```
+
+CSS selectors are also not capable of quering parent elements:
+
+```html
+<html>
+    <body>
+        <!-- ... -->
+                    <div>
+                        <!-- you cannot query this div by its child's ID -->
+                        <div>
+                            <div id="query-my-parent"></div>
+                        </div>
+                    </div>
+        <!-- ... -->
+    </body>
+</html>
+```
+
+To address these limitations, TestCafe introduces selector functions (or simply *selectors*).
```

Comment:
Maybe we can rephrase the end of this sentence without `(or simply *selectors)*`?